src/providers/lemonfox/api.ts

```diff
@@ -0,0 +1,27 @@
+import { ProviderAPIConfig } from '../types';
+
+const LemonfoxAIAPIConfig: ProviderAPIConfig = {
```

Comment:
would recommend keeping the naming of this provider standard across folder naming, and object names.
Either rename the folder to `lemonfox-ai` like `novita-ai` or rename the methods and variables to `lemonfox` as in `LemonfoxAPIConfig`

Reply:
done